StRoot/StMuDSTMaker/COMMON/StMuFstUtil.cxx

```diff
@@ -64,10 +68,32 @@ void StMuFstUtil::fillFst(StFstHitCollection* fstcol,StMuFstCollection* muFst)
 } // fillFst
 
 void StMuFstUtil::fillMuFstHits(StMuFstCollection* muFst,
-        StFstHitCollection* fstcol)
+        StFstHitCollection* fstcol,
+        StFstCollection *fstCollectionPtr)
 {
-    LOG_INFO << "fillMuFstHits" << endm;
-
+    //fill FST Raw Hits
+    if(fstCollectionPtr) {
+        LOG_INFO << "fillMuFstHits (raw hits)" << endm;
```

Comment:
No need to print this out on every event.  LOG_DEBUG would be more than sufficient...

Reply:
I changed back to print out only for FST hits to be consistent with other forward detector and used LOG_DEBUG for FST raw hits.